build.gradle.kts

```diff
@@ -94,12 +96,25 @@ subprojects {
         }
     }
 
+    val compileJavaVersion = JavaLanguageVersion.of(17)
+
+    pluginManager.withPlugin("org.jetbrains.kotlin.jvm") {
+        configure<KotlinJvmProjectExtension> {
+            compilerOptions {
+                jvmTarget = JvmTarget.JVM_11
+            }
+            jvmToolchain {
+                languageVersion = compileJavaVersion
+            }
+        }
+    }
+
     tasks.withType<JavaCompile>().configureEach {
-        sourceCompatibility = JavaVersion.VERSION_1_8.toString()
-        targetCompatibility = JavaVersion.VERSION_1_8.toString()
+        sourceCompatibility = JavaVersion.VERSION_11.toString()
+        targetCompatibility = JavaVersion.VERSION_11.toString()
```

Comment:
This might be a big deal, I don't know. I doubt many people are using Java 8 to run Gradle but it's worth noting in the changelog at least